index_deskop.html

```diff
@@ -0,0 +1,314 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="desktop.css"/>
+    <title>Desktop Version</title>
+  </head>
+  <body>
+    <div class="background">
+      <nav class=" flex-container-navigation">
+        <a class="flex-item flex-item-1 p-1-nav logo" href="#">My Logo</a>
+        <ul class="flex-item flex-item-2 top_nav_ul">
+          <li> <a href="#">Portfolio </a> </li>
+          <li> <a href="#">About </a> </li>
+          <li> <a href="#">Contact </a> </li>
+        </ul>
+        <a href="javascript:void(0)" class="not_show_bars p-1-nav" onclick="myFunction()"><i class="fa fa-bars"></i></a>
+      </nav>
+      <div class="content">
+      <main>
+        <h1>
+          I’m Waqas <br />
+          Glad to see you!
+        </h1>
+        <p class="description">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <h4 class="connect">LET’S CONNECT</h4>
+        <div>
+          <ul class="on_row">
+            <li>
+              <a href="#">
+                <img class="size" src="images/vector_1.png" alt="" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img class="size" src="images/vector_5.png" alt="" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img class="size" src="images/vector_2.png" alt=""
+              /></a>
+            </li>
+            <li>
+              <a href="#">
+                <img class="size" src="images/vector_3.png" alt=""
+              /></a>
```

Comment:
It is working fine with Me.
As show below in the following pictures 

![Screenshot 2022-04-29 at 23-23-15 Desktop Version](https://user-images.githubusercontent.com/13853450/166002167-a8a14903-d0d0-4c0a-b0db-7bacfdfb79fc.png)

![Screenshot 2022-04-29 at 23-23-39 Desktop Version](https://user-images.githubusercontent.com/13853450/166002220-d7d78cec-3c32-44b7-a319-4cae4d68cb64.png)

Reply:
It happens sometimes that the path you specified in your local machine can be followed by a specific live server (Since they have inbuilt packages that support this functionality) but not in other live servers, or even when deploying to Github pages. That is why you need to ensure the file names are as accurate as possible so they may be able to run on almost all servers, irrespective of their packages.